### DISCUSSION TITLE
News feed is not loading

### DISCUSSION THREAD

**Discussion by karthikasasanka** [community/community#21967] [New to GitHub]
<p>Hi</p>
<p>The news feed is not loading for my Github account</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/075b93fe128f1d1c0e22ef422a96ab71a90ff1dc.png" data-download-href="/uploads/short-url/135xWqA8WMcXLmdHB4OOIUldkja.png?dl=1" title="empty news feed" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/8085014/181067928-2161f6bc-f69a-4120-87fd-ca75dd77a1bc.png" alt="empty news feed" data-base62-sha1="135xWqA8WMcXLmdHB4OOIUldkja" width="448" height="235" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/075b93fe128f1d1c0e22ef422a96ab71a90ff1dc_2_448x235.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/075b93fe128f1d1c0e22ef422a96ab71a90ff1dc_2_672x352.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/0/075b93fe128f1d1c0e22ef422a96ab71a90ff1dc_2_896x470.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/075b93fe128f1d1c0e22ef422a96ab71a90ff1dc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">empty news feed</span><span class="informations">898×472 28 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>is there any setting that I should change to get back the feed?</p>
<p>Kindly let me know.</p>

**Comment by nethgato** [comment]
<p>Hey <a class="mention" href="/karthikasasanka">@karthikasasanka</a> o/</p>
<p>There’s currently an internal-only tool which will clear a users’ activity feed. We typically suggest this if we see a feed kicking out a :unicorn: but it might be appropriate, here.</p>
<p>Before I take any action which might clear your feed history, can you please confirm this is still going on?</p>
<p>I have also checked to make sure, and you’re certainly following some folks and I’d expect there to be <em>some</em> activity.</p>
<p>So again, lmk if it’s still ongoing, and I can kick it in the pants for you.</p>

**Comment by karthikasasanka** [comment]
<p>hi,</p>
<p>until yesterday, i was able to see updates in feed. but today morning when i checked feed is blank.</p>
<p>please provide a fix.</p>
<p>Regards,<br>
Kartheek</p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey Kartheek,</p>
<p>Thanks for following up! I appreciate the :+1: and I’ve gone ahead and cleared your activity feed on the backend.</p>
<p>I would recommend logging in from an Incognito or Private Browsing window and perhaps test by following a very active repository, or contributor.</p>
<p>If you continue to have problems, I’d be curious to know if there is a specific repo or individual that you’re following where you’re not seeing updates.</p>
<p>I really hope this worked, though!! :crossed_fingers:</p>

**Reply by Apollo-Blaze** [reply]
> Hey Kartheek,
> 
> Thanks for following up! I appreciate the 👍 and I’ve gone ahead and cleared your activity feed on the backend.
> 
> I would recommend logging in from an Incognito or Private Browsing window and perhaps test by following a very active repository, or contributor.
> 
> If you continue to have problems, I’d be curious to know if there is a specific repo or individual that you’re following where you’re not seeing updates.
> 
> I really hope this worked, though!! 🤞

Hey I have the same issue please fix it

**Comment by karthikasasanka** [comment]
<p>its working now</p>
<p>able to see feed</p>
<p>thank you</p>

**Comment by nethgato** [comment]
<p>That’s wonderful! Thank you so much for following up <a class="mention" href="/karthikasasanka">@karthikasasanka</a></p>
<p>It’s unfortunately a rather destructive action, which will remove historic feed data. We are working internally on a broader fix as I’m positive you are not the only one that’s had this experience.</p>
<p>The solution is also not sustainable. For all impacted, we wouldn’t be able to manually go and take that action. From what I’ve seen, there is discussion around exposing that option (completely clear your activity feed) to users, but it hasn’t been decided yet.</p>
<p>If there is anything relevant that comes from that conversation, I’ll come back here to post.</p>

**Comment by Spounka** [comment]
<p>Having the same issue here since a while now<br>
How can i fix it ?</p>
<p><img src="https://user-images.githubusercontent.com/30209821/181067937-d1abe0e3-2246-40d7-8e35-d8ce65e093e6.jpeg" alt="NewsFeed" data-base62-sha1="eNIYGtJNZXQuIVPL0aqB4QEAU8Y" width="672" height="177"></p>

**Comment by nethgato** [comment]
<p>Hi <a class="mention" href="/spounka">@Spounka</a> !</p>
<p>Unfortunately, there is no self-serve option to fix your activity feed. In fact, there is a destructive action that I can take, which should fix activity going forward.</p>
<p>So similarly to how I communicated this with Karthik above, I would love to get your :+1: before taking that action.</p>
<p>Please, if you could @ mention me in this thread for the go-ahead and I can take that action for you.</p>

**Comment by rajivnayanc** [comment]
<p>Hey <a class="mention" href="/nethgato">@nethgato</a> ,</p>
<p>Since yesterday, I am not able to see the news feed. It is blank.<br>
<img src="https://user-images.githubusercontent.com/31794134/181067941-6d0d5406-4d6e-44ad-ace7-4952f9c5cc69.png" alt="image" data-base62-sha1="sQbLINAWFNcDF3Ll78ezQFf4Pp6" width="636" height="414"></p>
<p>It would be great if you could provide a fix.<br>
Thanks,<br>
Rajiv Nayan Choubey</p>

**Comment by nethgato** [comment]
<p>Hi <a class="mention" href="/rajivnayanc">@rajivnayanc</a> :wave:</p>
<p>I’ve gone ahead and taken this action. You should see your feed start to populate again.</p>
<p>Just a reminder to anyone in this thread, the action that needs to be taken is a <strong>destructive action</strong> but should allow for your feed to populate normally going forward.</p>
<p>Please let me know if you continue to run into any trouble.</p>

**Comment by rajivnayanc** [comment]
<p>Thanks <a class="mention" href="/nethgato">@nethgato</a>, it works now. :slightly_smiling_face:</p>

**Comment by chinmay314** [comment]
<p>I am also facing the same issue. How can I reach staff in order to have the same action committed on my account?</p>

**Comment by logical-luke** [comment]
<p>Hi <a class="mention" href="/nethgato">@nethgato</a><br>
I do have same issue with my news feed.<br>
Is it possible to have the same <strong>destructive action</strong> done on my news feed history?</p>

**Comment by nethgato** [comment]
<p>Hey <a class="mention" href="/logical-luke">@logical-luke</a> :wave:</p>
<p>This is done! Your activity feed should now populate going forward, without issue. :crossed_fingers:</p>

**Comment by mjbcopland** [comment]
<p><a class="mention" href="/nethgato">@nethgato</a> Can you reset my activity feed too? I’ve attached a partially redacted screenshot showing my issue.</p>
<p>Strangely there is some activity in the feed but unfortunately it isn’t the issues and pull requests that I’m interested in. Even stranger is that these things <em>do</em> appear in the recent activity sidebar, showing that there is activity that I would expect to be seeing in the main feed.</p>
<p>I acknowledge that this is destructive and I am happy to lose what little activity I can currently see if it means that everything starts working again going forward.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/7/e775db73529fc6b7994291fef2a838fa69a2cde8.png" data-download-href="/uploads/short-url/x1ANP7polE56pkRvyCF6jAhM7ck.png?dl=1" title="github.com_ (1)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/16596696/181067952-3416a2da-d979-430a-a80a-cf521a39afb5.png" alt="github.com_ (1)" data-base62-sha1="x1ANP7polE56pkRvyCF6jAhM7ck" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e775db73529fc6b7994291fef2a838fa69a2cde8_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e775db73529fc6b7994291fef2a838fa69a2cde8_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e775db73529fc6b7994291fef2a838fa69a2cde8_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e775db73529fc6b7994291fef2a838fa69a2cde8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github.com_ (1)</span><span class="informations">2560×1440 366 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by nethgato** [comment]
<p>Hey <a class="mention" href="/mjbcopland">@mjbcopland</a> :wave:</p>
<p>I’ve gone ahead and taken care of that for you. Please let us know if your feed doesn’t get kicked back into gear.</p>
<p>Cheers! :bow:</p>

**Comment by carage34** [comment]
<p>Hello,</p>
<p>My github feed is not updated anymore.<br>
Can you tell me if i you can do something by your side ?</p>
<p>Regards.</p>

**Comment by nethgato** [comment]
<p>Hey <a class="mention" href="/carage34">@carage34</a> :wave:</p>
<p>I certainly can. However as discussed above, the option I have is a destructive action, which will completely clear your news feed. The action taken typically resolves the problem going forward from the time of that action, and will not restore your news feed history.</p>
<p>If you would like me to take that action, please reply and give the :+1:</p>
<p>Cheers!</p>

**Comment by carage34** [comment]
<p>Ok i’m fine with that. :+1:</p>
<p>Regards</p>

**Comment by nethgato** [comment]
<p>Gotchu <a class="mention" href="/carage34">@carage34</a> – go ahead and do a hard refresh, and check your activity going forward. :bow:</p>

**Comment by KevinNoffc** [comment]
@nethgato Could you clear my feed too? I have the same issue with the feed showing nothing..

**Reply by nethgato** [reply]
Oh hi there! I'm no longer at GitHub and not able to do that for you, but I'm sure another staff member can help you.

**Comment by demuux** [comment]
Hey there! Got the same problem. Who can I ask to fix it?

**Comment by JeanTheFungle** [comment]
it dosent work for me today

**Comment by KevinBrack** [comment]
@nethgato Please nuke my feed. I went ahead and gave you a bunch of thumbs up 👍. Thanks for taking your time to help users with this issue!

**Comment by ezkrg** [comment]
@nethgato my feed showing trending and recommended only or showing old / outdated news after reset filter to defaults. help would be grateful!

**Comment by nvm1** [comment]
@nethgato , same problem here.. Any chances you can help with the issue?
